packages/web-js-sdk/src/enhancers/withFingerprint/helpers.ts

```diff
@@ -63,25 +69,47 @@ const getFPFromStorage = (returnExpired = false): FingerprintObject => {
  * NOTE: Using fingerprintJS data has cost, use considerably.
  * @param fpKey FingerprintJS API key
  */
-export const ensureFingerprintIds = async (fpKey: string) => {
+export const ensureFingerprintIds = async (
+  fpKey: string,
+  baseUrl = FP_EP_URL,
+) => {
   try {
     if (getFPFromStorage()) {
       // FP is already in storage, no need to
       return;
     }
 
     const sessionId = generateUUID();
-    const agentP = load({ apiKey: fpKey, endpoint: FP_EP_URL });
+
+    const endpointUrl = new URL(baseUrl);
+    endpointUrl.pathname = FP_CF_ENDPOINT_PATH;
+
+    const patterUrl = new URL(baseUrl);
+    patterUrl.pathname = FP_CF_SCRIPT_PATH;
+    const scriptUrlPattern =
+      patterUrl.toString() +
+      '?apiKey=<apiKey>&version=<version>&loaderVersion=<loaderVersion>';
```

Comment:
I'm not too familiar with this pattern, what are those query params? CF thing?

Reply:
Fingerprint library replace them with the correct values when performing the call.
The CF thing is only the `path` part